src/AutoPilotPlugins/PX4/SensorsSetup.qml

```diff
@@ -83,10 +83,22 @@ Item {
         "ROTATION_ROLL_270_YAW_135",
         "ROTATION_PITCH_90",
         "ROTATION_PITCH_270",
-        "ROTATION_ROLL_270_YAW_270",
+        "ROTATION_PITCH_180_YAW_90",
+        "ROTATION_PITCH_180_YAW_270",
+        "ROTATION_ROLL_90_PITCH_90",
+        "ROTATION_ROLL_180_PITCH_90",
+        "ROTATION_ROLL_270_PITCH_90",
+        "ROTATION_ROLL_90_PITCH_180",
+        "ROTATION_ROLL_270_PITCH_180",
+        "ROTATION_ROLL_90_PITCH_270",
         "ROTATION_ROLL_180_PITCH_270",
-        "ROTATION_PITCH_90_YAW_180",
-        "ROTATION_ROLL_90_PITCH_90"
+        "ROTATION_ROLL_270_PITCH_270",
+        "ROTATION_ROLL_90_PITCH_180_YAW_90",
+        "ROTATION_ROLL_90_YAW_270",
+        "ROTATION_ROLL_90_PITCH_68_YAW_293",
+        "ROTATION_PITCH_315",
+        "ROTATION_ROLL_90_PITCH_315",
+        "ROTATION_CUSTOM
```

Comment:
```suggestion
        "ROTATION_CUSTOM"
```

Reply:
Probably missing a quote?

Reply:
Yes. Pushed a fix. Thanks for spotting that!